src/build/arg/mod.rs

```diff
@@ -3429,16 +3429,13 @@ impl<'help> Arg<'help> {
     ///     ]);
     ///
     /// assert!(res.is_err());
-    /// assert_eq!(res.unwrap_err().kind, ErrorKind::EmptyValue);
+    /// assert_eq!(res.unwrap_err().kind, ErrorKind::NoEquals);
     /// ```
     /// [`RequireEquals`]: ./enum.ArgSettings.html#variant.RequireEquals
-    /// [`ArgSettings::EmptyValues`]: ./enum.ArgSettings.html#variant.EmptyValues
-    /// [`ArgSettings::EmptyValues`]: ./enum.ArgSettings.html#variant.TakesValue
     #[inline]
     pub fn require_equals(self, r: bool) -> Self {
         if r {
-            self.unset_setting(ArgSettings::AllowEmptyValues)
-                .setting(ArgSettings::RequireEquals)
```

Comment:
You need to change line 3390. https://github.com/clap-rs/clap/pull/2399/files#diff-c6d33806f7e578137c01d6a9050010189cfecdaeb341f5101957ac517f5bfe10R3390

Reply:
Has been done in #2362